lms/static/js/spec/student_account/account_settings_fields_spec.js

```diff
@@ -46,6 +46,59 @@ define(['backbone',
                 );
             });
 
+            it("update time zone dropdown after country dropdown changes", function () {
+                requests = AjaxHelpers.requests(this);
+                var baseSelector = '.u-field-value > select';
+                var groups = baseSelector + '> optgroup';
+                var groupOptions = groups + '> option';
+                var normalOptions = baseSelector + '> option';
```

Comment:
Can you append `Selector` to each of these variables? `groupsSelector`, `groupOptionsSelector`, etc.